### PR TITLE
Animate to specific scroll controller offset

### PR DESCRIPTION
**Pull request by OlehRiepkin**

### CODE REVIEW

**Comment by tylervanderhoef** [comment]
Is there anything else we need to get this merged (aside from re-running checks)? This is something we desperately need. I don't want to maintain my own fork of this just because of this single PR not being merged, if it was just forgotten about or something.

Any update would be appreciated!

File: lib/src/day_view/day_view.dart [hunk 1]
```diff
@@ -217,6 +217,8 @@ class DayViewState<T extends Object?> extends State<DayView<T>> {
 
   late ScrollController _scrollController;
 
+  ScrollController get scrollController => _scrollController;
```
Comment (faiyaz-shaikh) [inline lib/src/day_view/day_view.dart:220]: @OlehRiepkin What is the use of this getter?

File: lib/src/week_view/week_view.dart [hunk 2]
```diff
@@ -216,6 +216,9 @@ class WeekViewState<T extends Object?> extends State<WeekView<T>> {
   EventController<T>? _controller;
 
   late ScrollController _scrollController;
+
+  ScrollController get scrollController => _scrollController;
```
Comment (faiyaz-shaikh) [inline lib/src/week_view/week_view.dart:220]: @OlehRiepkin ^^

File: lib/src/day_view/day_view.dart [hunk 3]
```diff
@@ -642,6 +642,19 @@ class DayViewState<T extends Object?> extends State<DayView<T>> {
...
+    Duration duration = const Duration(milliseconds: 200),
+    Curve curve = Curves.linear,
+  }) {
+    _scrollController.animateTo(
+      offset,
+      duration: duration,
+      curve: curve,
+    );
+  }
+
```
Comment (faiyaz-shaikh) [inline lib/src/day_view/day_view.dart:659]: @OlehRiepkin Where this method will be used.

**Comment by faiyaz-shaikh** [comment]
@OlehRiepkin Build is failing please check.

**Comment by faiyaz-shaikh** [comment]
Closing this PR as we have added code in PR #174 Thanks for Contribution @OlehRiepkin

**Pull request closed without merging**